hilti/runtime/CMakeLists.txt

```
@@ -90,9 +90,11 @@ target_compile_options(hilti-rt-debug-objects     PRIVATE "-UNDEBUG;-O0;-Wall")
 target_compile_definitions(hilti-rt-debug-objects PRIVATE "HILTI_RT_BUILD_TYPE_DEBUG")
 
 add_library(hilti-rt-tests-library-dummy1 SHARED src/tests/library-dummy.cc)
+target_link_libraries(hilti-rt-tests-library-dummy1 hilti-rt)
```

Let's revert this back to the inlined form to not break linking on freebsd.
```suggestion
target_include_directories(hilti-rt-tests-library-dummy1 PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

---

I had changed it because something else broke (don't remember right now what that was)

---

I suggested the explicit link via CMake dependencies here. At least on freebsd-12 this leads to duplicate symbols from jrx. Explicitly adding the include path seems to work around that.

This also fixes the inconsistency in how the two dummy libraries are linked.